app/Controller/ObjectsController.php

```diff
@@ -397,15 +404,25 @@ public function edit($id, $update_template_available=false, $onlyAddNewAttribute
         $templateData = $this->MispObject->resolveUpdatedTemplate($template, $object, $update_template_available);
         $this->set('updateable_attribute', $templateData['updateable_attribute']);
         $this->set('not_updateable_attribute', $templateData['not_updateable_attribute']);
-        if (isset($this->params['named']['revised_object'])) {
-            $revisedData = $this->MispObject->reviseObject($this->params['named']['revised_object'], $object, $template);
+        if (!empty($this->Cookie->read('object_being_created')) && !empty($this->params['named']['cur_object_tmp_uuid'])) {
```

Comment:
I think it should be better to save that data to session, not to cookie.

Reply:
Totally!